R/batch.R

```diff
@@ -170,5 +171,16 @@ decide_strategy <- function(splitting_strategy) {
 }
 
 verbose_set <- function(batchman.verbose) {
-  !identical(getOption('batchman.verbose'), FALSE) && isTRUE(batchman.verbose)
+  !identical(getOption("batchman.verbose"), FALSE) && isTRUE(batchman.verbose)
+}
+
+progress_bar <- function(batchman.verbose) {
+  if (isTRUE(batchman.verbose) && require(R6))
+    progress_estimated(num_batches, min_time = 3)
+  else
```

Comment:
You can remove these two lines, since `if` is an expression that returns null if the condition is false.